src/ONYXKEYS.ts

```diff
@@ -139,11 +139,11 @@ const ONYXKEYS = {
     /** The user's bank accounts */
     BANK_ACCOUNT_LIST: 'bankAccountList',
 
-    /** The user's credit cards (renamed from cardList) */
+    /** The user's credit / P2P cards */
```

Comment:
```suggestion
    /** The user's cash card and imported cards */
```

As discussed, P2P cards should go in fundList